src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx

```diff
@@ -474,3 +474,23 @@ Below are some of the most important attributes attached to [usage events](#data
     </tr>
   </tbody>
 </table>
+
+## Byte count estimate [#byte-count-estimate]
+
+One tool you can use to learn more about your usage is the function `bytecountestimate()` (for example, see [this query that uses it](#metrics-analysis)). 
+
+Some important points to know about using `bytecountestimate()`:
+
+* Because it returns all data it finds, and some of that data doesn't count towards your ingest and billing, it's only an estimate of your actual usage. For that reason, it's mostly useful for researching the data reported by various sources, or for understanding the upper ceiling of data reported. 
+* We recommend using it mainly for querying time ranges under 24 hours, and, if desired, using that to extrapolate. This is because it's a function that requires querying a lot of data, and because it won't find data that passes retention boundaries. 
+
+For more about estimating usage, see [our blog post about estimating usage](https://newrelic.com/blog/nerdlog/estimate-data-cost).
+
+Here are some technical details about how `bytecountestimate()` works: 
+
+* There are several types of data that [don't count towards ingest](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#usage-calculation) but that is returned by the `bytecountestimate()` function. This will mean that this function will sometimes show you **more** data than actually is counted towards your usage. It may rarely show a lower amount but, if it does, it will be only slightly lower. 
```

Comment:
```suggestion
* There are several types of data that [don't count towards ingest](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#usage-calculation) but that is returned by the `bytecountestimate()` function. This means that this function sometimes shows **more** data than actually is counted towards your usage. It may rarely show a lower amount but, if it does, it will be only slightly lower. 
```